<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newDiagramDialog</class>
 <widget class="QDialog" name="newDiagramDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Нова діаграма</string>
  </property>
  <property name="windowIcon">
   <iconset resource="v_resources.qrc">
    <normaloff>:/v_res/Resources/icons/document_16.png</normaloff>:/v_res/Resources/icons/document_16.png</iconset>
  </property>
  <property name="locale">
   <locale language="Ukrainian" country="Ukraine"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayoutDialog">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Назва:</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>nameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Тип:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>typeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="text">
          <string>NewDiagram</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="typeComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="modelColumn">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="parametersGroup">
         <property name="title">
          <string>Параметри</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayoutParams" rowstretch="0,0,0">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="vertex1xSpin">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>-5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="vertex1ySpin">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>-5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="diagonalVertex1Label">
            <property name="text">
             <string>Мінімальні значення в контейнері (x, y, z):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="vertex1zSpin">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>4.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>-5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="diagonalVertex2Label">
            <property name="text">
             <string>Максимальні значення в контейнері (x, y, z):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="vertex2zSpin">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-4.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="vertex2xSpin">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-4.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="vertex2ySpin">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-4.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
         <zorder>vertex1xSpin</zorder>
         <zorder>vertex1ySpin</zorder>
         <zorder>vertex1zSpin</zorder>
         <zorder>diagonalVertex2Label</zorder>
         <zorder>vertex2xSpin</zorder>
         <zorder>vertex2ySpin</zorder>
         <zorder>vertex2zSpin</zorder>
         <zorder>diagonalVertex1Label</zorder>
         <zorder>verticalSpacer</zorder>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>vertex1xSpin</tabstop>
  <tabstop>vertex1ySpin</tabstop>
  <tabstop>vertex1zSpin</tabstop>
  <tabstop>vertex2xSpin</tabstop>
  <tabstop>vertex2ySpin</tabstop>
  <tabstop>vertex2zSpin</tabstop>
 </tabstops>
 <resources>
  <include location="v_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>newDiagramDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>newDiagramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
